joomla/joomla-cms
# [5.0] order site/admin menus in admin ui

The site and admin menus displayed in the admin interface sidebar are hard coded to be displayed by id desc

This PR allows the order to be changed using drag and drop from com_menus&view=menus

Pull Request for Issue #38143 

When thinking about documentation for this PR I concluded that this is NOT a new feature but its more of a bug that you couldnt sort them before just as you can sort everything else.

### To test
Clean install from the prebuilt package in this PR
Create multiple site and admin menus (they dont need any menu items in them for the prupose of this test)
Drag and drop to change the order
Refresh the page and you will see the changes take effect in the sidebar

Update an existing test site using the prebuilt package in this pr
Repeat the steps above.

![menus](https://github.com/joomla/joomla-cms/assets/1296369/55ab4f17-201e-41c8-a726-c2c3fcb1a3ea)


thanks @HLeithner
